omp/tracer_omp.sh.in

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+exec_prefix=@prefix@
+libdir=@libdir@
+pkglibdir=$libdir/@PACKAGE@
+prefix=@prefix@
+datarootdir=@datarootdir@
+datadir=@datadir@
+
+display_help() {
+    echo "Usage:"
+    echo " $(basename $0) [options] [--] <application> <application-arguments>"
+    echo "  --help                        Show this screen"
+    echo "  --version                     Print the version string"
+    exit 1
+}
+
+display_version() {
+    cat $datadir/version
+    exit 1
+}
+
+while true; do
+  case "$1" in
+    --help           ) display_help; exit;;
```

Comment:
Missing option to trace Intel specific callbacks.

Reply:
Fixed by #f595184

Reply:
Fixed by https://github.com/argonne-lcf/THAPI/pull/33/commits/cfbef8db8a5c9abf4831447f4f56de886d70542c